cmd/up_test.go

```diff
@@ -194,3 +195,28 @@ func TestBuildSecret(t *testing.T) {
 		t.Errorf("wrong data")
 	}
 }
+
+func TestDump(t *testing.T) {
+	objs := []*unstructured.Unstructured{}
+	obj := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"foo": "bar",
+			"baz": map[string]string{
+				"1": "2",
+			},
+		},
+	}
+	objs = append(objs, obj)
+	var buf bytes.Buffer
+
+	err := dump(&buf, "yaml", objs)
+	if err != nil {
+		t.Error(err)
+	}
+	output := buf.String()
+	t.Log("output is", output)
+	if !strings.Contains(output, "foo") || !strings.Contains(output, "baz") {
```

Comment:
I would create test yaml and json output files for this test data and test it matches exactly